collectd_cuda/collectd_plugin.py

```diff
@@ -2,12 +2,19 @@
 
 import collectd
 import subprocess
+import numbers
 import xml.etree.ElementTree as ET
 
 
 def configure(conf):
         collectd.info('Configured with')
 
+# Checks for N/A or ERR! readings and returns 0.0
+def isvalid(reading):
```

Comment:
`isvalid` doesn't seem to correctly describe this function? Perhaps call it something like `normalize`?

Reply:
I agree, at the time it was the only name i could come up with.. I've changed the function to normalize.